project/Versions.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Properties
  */
 object Versions {
 
-  val crossScala = Seq("2.11.5", "2.10.4")
+  val crossScala = Seq("2.11.6", "2.10.5")
```

Comment:
Why would you want 2.11.6 *and* 2.11.5 as well as 2.10.5 *and* 2.10.4? They are binary compatible.